src/codegeneration/InstantiateModuleTransformer.js

```diff
@@ -174,8 +174,11 @@ class InsertBindingAssignmentTransformer extends ScopeTransformer {
  */
 export class InstantiateModuleTransformer extends ModuleTransformer {
 
-  constructor(identifierGenerator) {
-    super(identifierGenerator);
+  constructor(identifierGenerator, reporter, options) {
+    super(identifierGenerator, reporter, options);
+
+    this.anonymousModule =
+        options && !options.bundle && options.moduleName !== true;
```

Comment:
Can you check the caller?

The short term hack is to mark the option as optional:

```js
constructor(identifierGenerator, reporter, options = undefined) {
```